app/main/views/briefs.py

```diff
@@ -151,12 +149,43 @@ def update_brief_add_user(brief_id, user):
     return jsonify(briefs=brief.serialize(with_users=True)), 200
 
 
+@main.route('/briefs/<int:brief_id>/suppliers/<int:supplier_code>', methods=['PUT'])
+def update_brief_add_rfx_seller(brief_id, supplier_code):
```

Comment:
If this isn't RFX specific, it might be worthwhile renaming this function and the delete one to something like `add_seller_to_brief_data`.

Reply:
I believe these two methods must(should) be RFX specific. Within the JSON data blob for briefs, non-RFX invited sellers are stored as an array (sellerEmail for individuals, or sellerEmailList for multiple sellers), whereas RFX invited sellers are stored as a nested dictionary. Within the admin front end buyers.py they are all collated under the update_brief method.

Reply:
If this is RFX specific, we may need to check the brief's lot and abort if a non-RFX brief is passed to this route.